cmd/nvidia-container-runtime-hook/container_config.go

```diff
@@ -339,6 +354,7 @@ func getNvidiaConfig(hookConfig *HookConfig, image image.CUDA, mounts []Mount, p
 		Devices:            devices,
 		MigConfigDevices:   migConfigDevices,
 		MigMonitorDevices:  migMonitorDevices,
+		ImexChannels:       imexChannels,
```

Comment:
Could also just use:
```suggestion
		ImexChannels:       image.Getenv(envNVImexChannels),
```
directly here.